lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
The problem is that the base reporter does not trigger if the event name is `skipped`, see https://github.com/webdriverio/webdriverio/blob/master/lib/utils/BaseReporter.js#L114